packages/schematics/angular/application/files/module-files/src/app/app.component.ts.template

```diff
@@ -15,5 +15,5 @@ import { Component } from '@angular/core';
   styleUrl: './app.component.<%= style %>'<% } %>
 })
 export class AppComponent {
-  title = '<%= name %>';
+  title = signal('<%= name %>');
```

Comment:
Is it logical for this to be considered a signal, given that it's just a static string?
PS: In case we go forward with this we need to update of a couple of other templates.

Reply:
I am not too strongly about this, but I wonder if it would send the wrong "signal", that anything referenced in the template needs to be a `signal`.

I'll defer to @alxhub. An alternative for correctness maybe I could see this title having the `readonly` modifier added.